conan/cli/commands/graph.py

```diff
@@ -109,12 +115,15 @@ def graph_build_order_merge(conan_api, parser, subparser, *args):
     Merge more than 1 build-order file.
     """
     subparser.add_argument("--file", nargs="?", action="append", help="Files to be merged")
+    subparser.add_argument("--order", choices=['configuration'],
```

Comment:
```suggestion
    subparser.add_argument("--order", choices=['configuration', 'revision'], default='revision'
```
Maybe less confussion so that it shows up in the help strings?